IPython/core/magics/pylab.py

```diff
@@ -42,6 +42,8 @@ class PylabMagics(Magics):
     @skip_doctest
     @line_magic
     @magic_arguments.magic_arguments()
+    @magic_arguments.argument('-l', '--list', default=False, action='store_true',
```

Comment:
I think `'store_true'` makes it clear enough what's going on. All the examples in the argparse docs using store_true don't bother with a default.